components/dashboard/src/projects/Project.tsx

```diff
@@ -184,6 +188,20 @@ export default function () {
         return date ? dayjs(date).fromNow() : "";
     };
 
+    const resumePrebuilds = async () => {
+        if (!project) {
+            return;
+        }
+        try {
+            const response = await getGitpodService().server.triggerPrebuild(project.id, null);
+            setIsConsideredInactive(false);
+            setIsResuming(false);
+            history.push(`/${!!team ? "t/" + team.slug : "projects"}/${projectSlug}/${response.prebuildId}`);
+        } catch (error) {
+            console.error(error);
+        }
```

Comment:
Nit: The `setIsResuming(false)` should also (or especially) be called when errors happen, right? Maybe move it into a `finally`?

```suggestion
            history.push(`/${!!team ? "t/" + team.slug : "projects"}/${projectSlug}/${response.prebuildId}`);
        } catch (error) {
            console.error(error);
        } finally {
            setIsResuming(false);
        }
```